graphql-client-support/src/main/java/com/expediagroup/streamplatform/streamregistry/graphql/client/LongTypeAdapter.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.streamplatform.streamregistry.graphql.client;
+
+
+import java.math.BigDecimal;
+
+import com.apollographql.apollo.response.CustomTypeAdapter;
+import com.apollographql.apollo.response.CustomTypeValue;
+
+public class LongTypeAdapter implements CustomTypeAdapter<Object> {
```

Comment:
Could you put this in the `graphql-client-support` module along with the existing `ObjectNodeTypeAdapter`?

Reply:
Hi, so it's exactly in that module already, maybe the package is incorrect?

Reply:
Should this implement CustomTypeAdapter<Long> rather than object?